testsuite/tests/grammar/indent_trivia/expected_concrete_syntax.lkt

```diff
@@ -1,4 +1,4 @@
-grammar None_grammar {
+grammar FooGrammar {
```

Comment:
foo_grammar would be better if we keep capitalized names for types

Reply:
Will fix.